src/editor/Core/ConfigurationDefaults.re

```diff
@@ -43,6 +43,7 @@ let getDefaultConfigString = configName =>
 {
     "bindings": [
         { "key": "<C-P>", "command": "quickOpen.open", "when": [["editorTextFocus"]] },
+        { "key": "<C-TAB>", "command": "quickOpen.openFiles", "when": [["editorTextFocus"]] },
```

Comment:
When I tested locally, I had some problems with `"<C-TAB>"` getting picked up correctly - due to bugs in our input code. Working on a fix for those.